web-ui/src/main/resources/catalog/views/default/less/gn_card_default.less

```diff
@@ -0,0 +1,567 @@
+@import "../../../style/gn_bootstrap.less";
+@import "gn_variables_default.less";
+
+// status
+.gn-status, .gn-workflow-status {
+  position: relative;
+  float: left;
```

Comment:
It looks like this has some side effects in the editor ![image](https://user-images.githubusercontent.com/1701393/111771084-865da000-88ab-11eb-9f17-5fc6b4197767.png)

and in record view (fix by) https://github.com/geonetwork/core-geonetwork/pull/5524 

@MichelGabriel can you check for the editor ?

Reply:
It's used in the editor? Ok, going to check

Reply:
I see, that block in the editor automatically got the class `gn-status`, which messes up styling